packages/component-library/styles/fonts.scss

```diff
@@ -1,5 +1,13 @@
 @import "~@kaizen/hosted-assets/index";
 
+// =============== TIEMPOS ===============
+@font-face {
+  font-family: "Tiempos Headline";
+  src: url(asset-url("fonts/tiempos/tiempos-headline-bold.woff2")),
```

Comment:
Used Bold instead of Black because it looked incorrect. 
Using Tiempos Bold: 
![Screen Shot 2021-02-18 at 4 49 22 pm](https://user-images.githubusercontent.com/1621182/108311772-b3b22200-7209-11eb-90f2-47ee0551c832.png)

Using Tiempos Black 
![Screen Shot 2021-02-18 at 4 46 30 pm](https://user-images.githubusercontent.com/1621182/108311780-b6ad1280-7209-11eb-8214-d90d1fd998e9.png)

Reply:
FYI @katsambarcus ☝️